packages/bpk-docs/src/pages/NativeCardPage/NativeCardPage.js

```diff
@@ -0,0 +1,99 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import readme from './../../../../../native/packages/react-native-bpk-component-card/readme.md';
+
+import iosScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/ios/default.png';
+import androidScreenshotDefault from './../../../../../native/packages/react-native-bpk-component-card/screenshots/android/default.png';
```

Comment:
For now, as we aren't officially supporting Android yet, we should use the android_empty image:
`import androidEmpty from './../../static/android_empty.svg';`